packages/desktop-client/src/components/accounts/Account.tsx

```
@@ -828,100 +693,91 @@ class AccountInternal extends PureComponent<
       case 'close':
-        void this.props.dispatch(openAccountCloseModal({ accountId }));
+        void dispatch(openAccountCloseModal({ accountId }));
         break;
       case 'reopen':
-        this.props.onReopenAccount(accountId);
+        props.onReopenAccount(accountId);
         break;
       case 'export':
-        const accountName = this.getAccountTitle(account, accountId);
-        void this.onExport(accountName);
+        const accountName = getAccountTitle(account, accountId);
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)